x-pack/test_serverless/shared/services/bsearch_secure.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// NOTE: This is pretty much a copy/paste from test/common/services/bsearch.ts but with the ability
+// to provide custom auth
+
+import expect from '@kbn/expect';
+import { GenericFtrService } from '@kbn/test';
+import request from 'superagent';
+import type SuperTest from 'supertest';
+import { IEsSearchResponse } from '@kbn/data-plugin/common';
+import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
+import { BFETCH_ROUTE_VERSION_LATEST } from '@kbn/bfetch-plugin/common';
+import { FtrProviderContext } from '../../functional/ftr_provider_context';
+
+const parseBfetchResponse = (resp: request.Response): Array<Record<string, any>> => {
+  return resp.text
+    .trim()
+    .split('\n')
+    .map((item) => JSON.parse(item));
+};
+
+const getSpaceUrlPrefix = (spaceId?: string): string => {
+  return spaceId && spaceId !== 'default' ? `/s/${spaceId}` : ``;
+};
```

Comment:
question: this FTR service is located in `x-pack/test_serverless` path and assumed to be adopted for serverless env (including real MKI). Afaik we don't support spaces in serverless or support was added?

Reply:
Yes, serverless has not supported spaces yet. I should remove space prefix.

Reply:
@angorayc is @dmlemeshko 's comment above addressed?  Looks like it might be.

Reply:
yes @wayneseymour , `getSpaceUrlPrefix ` has been removed.